source/scripts/model/Monster.js

```diff
@@ -34,16 +48,7 @@ export default class Monster {
         this.animation = false
 
         if(this.phase == true) {
```

Comment:
Basically, so we can have monsters that move slower than others (ie, on every Nth turn or every turn) we need to move this phase logic into the movement function.